src/main/java/seedu/address/logic/commands/ClientEditCommand.java

```diff
@@ -145,6 +151,9 @@ public boolean equals(Object other) {
         private Timezone timezone;
         private ContractExpiryDate contractExpiryDate;
 
+        /**
+         * Empty constructor.
+         */
```

Comment:
should this be following the same commenting standard? 

"Constructs an EditClientDescriptor. It is an empty constructor that takes in no arguments." or something like that?

Reply:
done